apps/web/src/app/dashboard/[name]/settings/layout.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import Link from "next/link";
+import { usePathname, useRouter } from "next/navigation";
+import type { PropsWithChildren } from "react";
+
+import { cn } from "~/lib/utils";
+
+export default function Settings({ children }: PropsWithChildren) {
+  const pathname = usePathname();
+
+  return (
+    <div className="mx-auto w-full flex-1 px-6 py-8 sm:p-12">
+      <div className=" mx-auto w-full max-w-6xl">
+        <div className="flex h-full flex-col lg:flex-row">
+          <nav className="mr-2 w-full shrink-0 lg:w-1/5">
+            <div className="mt-1 hidden flex-col pr-2 lg:flex">
+              <Link href="/dashboard/stasiu/settings/general">
```

Comment:
`Link` here are static, shouldn't they be dynamic?

Reply:
I was writing it at 3 AM...